### DISCUSSION TITLE
SlingerPlayer URL to use when accessing remotely and not on LAN

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/SlingerPlayer#29] [Q&A] [answered]
In the manual it says if you want to connect when not on LAN just put the word 'wan' in front of the ipaddress of slinger with the port number (e.g.  "wan://IPAddressOfSlinger:PortNumber I.E. wan://192.169.232.22:65432")

why is the external ip address not needed (e.g. "http://67.8X.XXX.XXX:65432/urlbase/nameofslingbox") as one does when using other media players like VLC?
how does SlingerPlayer know where to look?

thanks

**Comment by GerryDazoo** [comment, marked as answer]
Probably a bad example in the doc but 192.169.232.22 is an external IP address, It's **192.168**.x.x that is a local subnet.
The wan://external_ip_address:external_port option tells SlingerPlayer to query the server so it can auto-populate the drop down list of all available Slingbox configurations. This capability is specific to SlingerPlayer/Slinger.
If you know which config you want to use you can supply it directly just like your VLC example.
